[PR] Person markup

philcable opened the pull request:
The related UC Projects output still needs to be customized - I'll tackle that in another PR.

The latest People Directory updates should also ship before this :)

Commit:
Fix the Grunt watch task so it doesn't run indefinitely

Commit:
Add person profile style guide

I'm being a bit lazy and using the content syndicate markup for the
related posts here since the styles are already in place for it.

Commit:
Adjust content section selectors

This makes things more general so that all content is centered and
constrained to the max width of 1188 pixels. Hopefully it's not
too general - I've looked through all the views I could think of
and didn't see any problems, but I could have missed something.

Commit:
Add person profile styles

Commit:
Update HTML output for single profile views

Relies on https://github.com/washingtonstateuniversity/WSU-People-Directory/pull/99

Commit:
Adjust spacing to meet PHPCS

jeremyfelt commented on wsu-people/person.php:
```diff
@@ -0,0 +1,129 @@
+
+		<hgroup>
+			<h1 class="article-title"><?php echo esc_html( $display['name'] ); ?></h1>
+			<span class="degree">MBA</span>
```
It looks like these didn't get replaced with dynamic content.

philcable replied:
Good catch, I'll get that fixed!

jeremyfelt reviewed (changes requested):
I left one inline comment that should be addressed.

This looks good, but did get me thinking. Is there an opportunity to use something like `get_template_part( 'person', $display['directory_view'] )` to provide the different layouts either as part of `wsu-people/person.php` or elsewhere in the plugin before that template is loaded? This could give the plugin the opportunity to have some defaults that can be specifically overwritten.

Commit:
Add custom `project` output to the person template

Commit:
Output degrees for a person

This relies on https://github.com/washingtonstateuniversity/WSU-People-Directory/pull/101

philcable commented:
I was thinking the same thing while building this out. Definitely a good idea, and I think it would be pretty straightforward to implement.

I also fibbed and added the University Center Projects output in this branch, so that bit will need a review too!

philcable commented:
@jeremyfelt #reviewmerge?

The UC Projects output is still using the Content Syndicate markup, which might not be a big deal, but seems lazy to me and should probably change before launch.

jeremyfelt commented:
It works as a start. We can see what it looks like and then adjust from there.
